genesyscloud/task_management_workbin/data_source_genesyscloud_task_management_workbin.go

```diff
@@ -21,7 +21,7 @@ import (
 // dataSourceTaskManagementWorkbinRead retrieves by name the id in question
 func dataSourceTaskManagementWorkbinRead(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
 	sdkConfig := meta.(*provider.ProviderMeta).ClientConfig
-	proxy := newTaskManagementWorkbinProxy(sdkConfig)
+	proxy := getTaskManagementWorkbinProxy(sdkConfig)
```

Comment:
Found a few spots where Prince was use doing a new to get the proxy instead of a getXXX